### PR TITLE
Enrich simplemodal's package.json(keywords/repository/license)

### PR DESCRIPTION
**Pull request by bmiwdmb**
#5422  
#5194

### CODE REVIEW

File: ajax/libs/simplemodal/package.json [hunk 1]
```diff
@@ -3,5 +3,22 @@
...
+    "wordpress",
+    "jquery",
+    "ui architect",
+    "web standards",
+    "usability",
+    "consulting",
+    "php",
+    "modal"
+  ],
+  "license": "MIT"
```
Comment (PeterDaveHello) [inline ajax/libs/simplemodal/package.json]: Thought there is MIT license in repo, I guess they are using GPL?

See https://github.com/ericmmartin/simplemodal/blob/master/bower.json

File: ajax/libs/simplemodal/package.json [hunk 2]
```diff
@@ -3,5 +3,22 @@
...
-  "homepage": "http://www.ericmmartin.com/projects/simplemodal/"
+  "homepage": "http://www.ericmmartin.com/projects/simplemodal/",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/ericmmartin/simplemodal"
+  },
+  "keywords": [
+    "eric martin",
+    "simplemodal",
+    "wordpress",
```
Comment (PeterDaveHello) [inline ajax/libs/simplemodal/package.json]: May I ask what's the reason why to add `wordpress`?

File: ajax/libs/simplemodal/package.json [hunk 3]
```diff
@@ -3,5 +3,22 @@
...
   "version": "1.4.4",
   "description": "SimpleModal is a lightweight jQuery Plugin which provides a powerful interface for modal dialog development.",
-  "homepage": "http://www.ericmmartin.com/projects/simplemodal/"
+  "homepage": "http://www.ericmmartin.com/projects/simplemodal/",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/ericmmartin/simplemodal"
+  },
+  "keywords": [
+    "eric martin",
```
Comment (PeterDaveHello) [inline ajax/libs/simplemodal/package.json]: I don't think adding author's name as keyword is a good idea.

File: ajax/libs/simplemodal/package.json [hunk 4]
```diff
@@ -3,5 +3,22 @@
...
+    "simplemodal",
+    "wordpress",
+    "jquery",
+    "ui architect",
+    "web standards",
+    "usability",
+    "consulting",
+    "php",
```
Comment (PeterDaveHello) [inline ajax/libs/simplemodal/package.json]: May I ask what's the reason why to add php?

**Comment by PeterDaveHello** [comment]
@cdnjs/intern please review this PR, especially the keywords, thanks.

**Comment by Piicksarn** [comment]
@bmiwdmb, you had better follow reliable source of keywords.
For example, `.json` files of library always include some keywords and other information.

I can not find any about "**php**" keyword, please discuss the source or make some correction in your commit. :)

File: ajax/libs/simplemodal/package.json (hunk 1 above)
Comment (bmiwdmb) [inline ajax/libs/simplemodal/package.json]: [simplemodal/License.txt](https://github.com/ericmmartin/simplemodal/blob/master/License.txt)

File: ajax/libs/simplemodal/package.json [hunk 5]
```diff
@@ -3,5 +3,18 @@
...
+    "type": "git",
+    "url": "https://github.com/ericmmartin/simplemodal"
+  },
+  "keywords": [
+    "simplemodal",
+    "jquery",
+    "framework"
+  ],
+  "license": [
+    "MIT",
```
Comment (PeterDaveHello) [inline ajax/libs/simplemodal/package.json]: I think we should remove MIT unless you can prove that they are using MIT.

**Comment by bmiwdmb** [comment]
fix completed, Please review the pull request!

**Comment by PeterDaveHello** [comment]
b08bd9a736f66d33226cb20111a76ca992a8f997 is unnecessary, please remove it and rebase with latest master branch.

**Comment by bmiwdmb** [comment]
finished

**Comment by PeterDaveHello** [comment]
Bingo, but you should squash them into only one commit, thanks.

**Comment by bmiwdmb** [comment]
fix completed, Please review the pull request!

**Comment by PeterDaveHello** [comment]
Seems you didn't rebase with our latest master branch, but it's fine in this case, I'll merge it.

**Pull request merged**